receiver/redisreceiver/redis_runnable_test.go

```diff
@@ -32,14 +33,14 @@ func TestRedisRunnable(t *testing.T) {
 	err = runner.Run()
 	require.Nil(t, err)
 	// + 6 because there are two keyspace entries each of which has three metrics
-	require.Equal(t, len(getDefaultRedisMetrics())+6, len(consumer.md.Metrics))
+	require.Equal(t, len(getDefaultRedisMetrics())+6, pdatautil.MetricCount(consumer.md))
 }
 
 type fakeMetricsConsumer struct {
```

Comment:
Use the `exportertest.Sink`

Reply:
done

Reply:
Is this really done?

Reply:
I think you wanted to mark the other comment as done

Reply:
Yes, I fixed another fakeMetricsConsumer. let me fix this one

Reply:
done this time